packages/field-plugin/src/createFieldPlugin/createFieldPlugin.ts

```diff
@@ -99,13 +99,13 @@ export const createFieldPlugin: CreateFieldPlugin = ({
 
   const cleanupHeightChangeListener = createHeightChangeListener(onHeightChange)
 
-  void initialize()
-
   const cleanupMessageListenerSideEffects = createPluginMessageListener(
     params.uid,
     messageCallbacks,
   )
 
+  void initialize()
```

Comment:
This is actually a real bug fix. Inside `createPluginMessageListener`, we `addEventListener("message", ..)`. We're actually supposed to add a event listener first, and then call the initialize function. That's the correct order.
However, in reality, it didn't introduce any bug because it's an asynchronous task for the container to receive the message and respond back to the field plugin. But anyway, this is the correct way, and doesn't break test code :)